googlecompute/windows_images.json

```diff
@@ -52,6 +52,12 @@
         "docker pull cirrusci/android-sdk:30-windowsservercore-2019",
         "docker pull python:3-windowsservercore"
       ]
+    },
+    {
+      "type": "windows-shell",
+      "inline": [
+        "powershell -Command \"Set-MpPreference -DisableRealtimeMonitoring 1\""
```

Comment:
Maybe move it to the top so it's disabled for Docker installation and pulls. Maybe it'll make it noticeably faster too. 🤔

Reply:
See https://github.com/cirruslabs/vm-images/pull/8/commits/cb7ba7f29416fe4c0bf6a016650e8af829cf6a42.